specutils/spectra/spectrum1d.py

```diff
@@ -776,3 +765,23 @@ def __repr__(self):
         result = "<Spectrum1D({})>".format(inner_str)
 
         return result
+
+
+class Spectrum(Spectrum1D):
+    '''
+    Adding this to 1.x so that people can update the name of the class in their
+    code without needing to wait for the 2.0 release or update their version pin.
+    '''
+    def __init__(self, *args, **kwargs):
+        if 'move_spectral_axis' in kwargs:
```

Comment:
I think this would have needed at least an `or 'spectral_axis_index' in kwargs` to ensure that is really covered; or get its own full check and warning as we just discussed.

Reply:
`spectral_axis_Index` is handled further down, I just had it indented one level too far. Fixed by #1230